ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -257,38 +258,98 @@ export default class StatusDropdown extends React.Component {
     alertStatus === 'investigating' ||
     (alertStatus !== status && this.isResolved(alertStatus));
 
+  getRemainingTime(firstDate, secondDate) {
+    const differenceInTime = Math.abs(secondDate - firstDate);
+    let differenceInDays = Math.ceil(differenceInTime / (1000 * 60 * 60 * 24));
+
+    // If weekday of date1 is bigger than date2 then it is a new week
+    // and we have to subtract the weekend days
+    if (firstDate.getUTCDay() > secondDate.getUTCDay()) {
+      differenceInDays -= 2;
+    } else {
+      differenceInDays -= 2 * Math.trunc(differenceInDays / 7);
+    }
+
+    let differenceInHours = Math.ceil(differenceInTime / (1000 * 60 * 60));
+    // If due date is Monday and today is Friday and we have to show the hours left,
+    // we have to subtract the weekend hours from the difference
+    if (firstDate.getUTCDay() === 5 && secondDate.getUTCDay() === 1) {
```

Comment:
```suggestion
    const shouldGetHoursLeft = (currentDate.getUTCDay() === friday && dueDate.getUTCDay() === monday);
    if (shouldGetHoursLeft) {
```

Reply:
This was addressed partially. I will mark this as resolved because I left a similar change request.